src/components/ProfileForm/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+
+import Input from 'components/common/Input';
+import SelectAvatar from 'components/SelectAvatar';
+import strings from 'locale';
+import { PROFILE_SHAPE } from 'constants/shapes';
+import styles from './styles';
+
+const {
+  PROFILE: { usernameLabel, firstNameLabel, lastNameLabel },
+} = strings;
+
+const ProfileForm = ({ profile }) => {
+  const [avatarData, setAvatarData] = useState('');
+  const { username, firstName, lastName, email } = profile;
+  return (
```

Comment:
a new line before the return, this gives more readability 🙂

Reply:
Thanks! This file has also been changed!